plugins/builtin/strategy/fixed-value/plugin/plugin.go

```diff
@@ -0,0 +1,117 @@
+package plugin
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/nomad-autoscaler/plugins"
+	"github.com/hashicorp/nomad-autoscaler/plugins/base"
+	"github.com/hashicorp/nomad-autoscaler/plugins/strategy"
+	"github.com/hashicorp/nomad-autoscaler/sdk"
+)
+
+const (
+	// pluginName is the unique name of the this plugin amongst strategy
+	// plugins.
+	pluginName = "fixed-value"
+
+	// These are the keys read from the RunRequest.Config map.
+	runConfigKeyFixed    = "fixed"
+
+	RADIX_10 = 10
+
+)
+
+var (
+	PluginID = plugins.PluginID{
+		Name:       pluginName,
+		PluginType: sdk.PluginTypeStrategy,
+	}
+
+	PluginConfig = &plugins.InternalPluginConfig{
+		Factory: func(l hclog.Logger) interface{} { return NewFixedValuePlugin(l) },
+	}
+
+	pluginInfo = &base.PluginInfo{
+		Name:       pluginName,
+		PluginType: sdk.PluginTypeStrategy,
+	}
+)
+
+// Assert that StrategyPlugin meets the strategy.Strategy interface.
+var _ strategy.Strategy = (*StrategyPlugin)(nil)
+
+// StrategyPlugin is the FixedValue implementation of the strategy.Strategy
+// interface.
+type StrategyPlugin struct {
+	config map[string]string
+	logger hclog.Logger
+}
+
+// NewFixedValuePlugin returns the FixedValue implementation of the
+// strategy.Strategy interface.
+func NewFixedValuePlugin(log hclog.Logger) strategy.Strategy {
+	return &StrategyPlugin{
+		logger: log,
+	}
+}
+
+// SetConfig satisfies the SetConfig function on the base.Base interface.
+func (s *StrategyPlugin) SetConfig(config map[string]string) error {
+	s.config = config
+	return nil
+}
+
+// PluginInfo satisfies the PluginInfo function on the base.Base interface.
+func (s *StrategyPlugin) PluginInfo() (*base.PluginInfo, error) {
+	return pluginInfo, nil
+}
+
+// Run satisfies the Run function on the strategy.Strategy interface.
+func (s *StrategyPlugin) Run(eval *sdk.ScalingCheckEvaluation, count int64) (*sdk.ScalingCheckEvaluation, error) {
+
+	// Read and parse target value from req.Config.
+	t := eval.Check.Strategy.Config[runConfigKeyFixed]
+	if t == "" {
+		return nil, fmt.Errorf("missing required field `fixed`")
+	}
+
+	fixed, err := strconv.ParseInt(t, RADIX_10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("invalid value for `fixed`: %v (%T)", t, t)
```

Comment:
Same comment as above regarding the formatting of the `fixed` const. In addition it seems `err` is only evaluated and not formatted in the return, maybe this would be helpful?